File: pets4life-api/BusinessObject/Models/Order.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace BusinessObject.Models
{
    public partial class Order
    {
        public Order()
        {
            OrderDetails = new HashSet<OrderDetail>();
        }

        public int OrderId { get; set; }
        public int? UserId { get; set; }
        public DateTime? OrderDate { get; set; }
        public decimal? TotalAmount { get; set; }
        public bool? IsPaid { get; set; }
        public string? ImagePayment { get; set; }

        [JsonIgnore] public virtual User? User { get; set; }
        [JsonIgnore] public virtual ICollection<OrderDetail> OrderDetails { get; set; }
    }
}

File: petcare/lib/model-petcare/order.dart
class Order {
  int? orderId;
  int? userId;
  String? orderDate;
  double? totalAmount;
  bool? isPaid;

  Order(
      {this.orderId,
      this.userId,
      this.orderDate,
      this.totalAmount,
      this.isPaid});

  Order.fromJson(Map<String, dynamic> json) {
    orderId = json['orderId'];
    userId = json['userId'];
    orderDate = json['orderDate'];
    totalAmount = json['totalAmount'];
    isPaid = json['isPaid'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['orderId'] = this.orderId;
    data['userId'] = this.userId;
    data['orderDate'] = this.orderDate;
    data['totalAmount'] = this.totalAmount;
    data['isPaid'] = this.isPaid;
    return data;
  }
}
